File: Benchmarks/enterprise/python/fastapi/testcode/benchmark_test_60263.py
# SPDX-License-Identifier: Apache-2.0
from fastapi import Request
import logging
from pydantic import BaseModel


class UserInput(BaseModel):
    payload: str = ''

async def BenchmarkTest60263(request: Request, req: UserInput):
    json_value = req.payload
    data = ' '.join(str(json_value).split())
    logging.info('User action: ' + str(data))
    return {"updated": True}

File: Benchmarks/enterprise/java/spring/testcode/BenchmarkTest60263.java
// SPDX-License-Identifier: Apache-2.0
import jakarta.servlet.http.HttpServletRequest;
import jakarta.servlet.http.HttpServletResponse;
import org.springframework.web.bind.annotation.*;

@RestController
public class BenchmarkTest60263 {

    @GetMapping("/BenchmarkTest60263")
    public void BenchmarkTest60263(HttpServletRequest request, HttpServletResponse response) throws Exception {
        String secretValue = "p4ssw0rd_test_xyz";
        String data = String.join(" ", secretValue.split("\\s+"));
        try (java.sql.Connection authConn = java.sql.DriverManager.getConnection(
                "jdbc:postgresql://db.svc.local/app", "appuser", data)) {
            response.getWriter().print("{\"auth\":\"ok\"}");
        }
    }
}
